ci/Jenkinsfile

```diff
@@ -223,6 +227,7 @@ pipeline {
                             expression { return caseList.contains(Case) }
                         }
                         steps {
+                            catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
                             script {
```

Comment:
This whole block (to L278) should be indented one more level now.

Reply:
Yeah thanks [sic]